README.md

```diff
@@ -5,101 +5,76 @@
 Azure IoT PCS CLI Overview
 ==========================
 
-Command Line Interface for deploying an IoT preconfigured solution into a
-user's subscription.
+Command Line Interface for deploying an [Azure IoT solution](https://www.azureiotsolutions.com) into a
+user's Azure subscription.
 
-This CLI has the ability to deploy two configurations of PCS solutions:
+An IoT Solution is open source implementation of a common IoT solution patterns that you can deploy to Azure using your subscription. Each solution combines custom code and Azure services to implement a specific IoT scenario or scenarios. You can customize any of the scenarios to meet your specific requirements. Visit https://www.azureiotsolutions.com for more details or to deploy using the GUI.

```

Comment:
* split line
* missing "accelerator"
* typo on "a common .. patterns"
* nit: in "... combine ... Azure services..." remove "Azure"
* nit: scenarios are not customizable, it's the architecture/solution that can be
* make url a clickable link